src/constants.js

```diff
@@ -71,6 +71,7 @@ module.exports = {
   },
 
   SortBy: {
+    DEFAULT: 'DESC',
```

Comment:
nit: prefer to have this somewhere else. this should just be the constant itself.

Reply:
yes, we will eventually have different sorting setup for different pages